test/unit/test_resource_container.py

```diff
@@ -1,82 +0,0 @@
-import pytest
```

Comment:
Is it possible to update these tests or add new ones for the new discoverer classes? I'm happy to follow up with another PR to add them if you don't want to muck about in pytest.